.github/workflows/release.yml

```
@@ -43,11 +43,19 @@ jobs:
-    # Upload the JAR as a release asset
+    # The published Maven artifact is now the thin jar; use the runnable "shaded" jar as the
+    # downloadable CLI release asset, keeping the familiar TestingBotTunnel-<version>.jar name.
+    - name: Prepare runnable jar for release
+      run: |
+        VERSION=${GITHUB_REF#refs/tags/v}
+        mkdir -p release-assets
+        cp target/TestingBotTunnel-${VERSION}-shaded.jar release-assets/TestingBotTunnel-${VERSION}.jar
+
+    # Upload the runnable JAR as a release asset
     - name: Upload JAR to GitHub Releases
       uses: softprops/action-gh-release@v1
```

actionlint (1.7.12)

[error] the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)